spec/lib/api/2.0/views-spec.js

```diff
@@ -0,0 +1,147 @@
+// Copyright 2015, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Views', function () {
+    var viewsProtocol;
+    var _;
+
+    before('start HTTP server', function () {
+        this.timeout(10000);
+        viewsProtocol = {
+            load: sinon.stub(),
+            getAll: sinon.stub(),
+            get: sinon.stub(),
+            put: sinon.stub(),
+            unlink: sinon.stub()
+        };
+
+        return helper.startServer([
+            dihelper.simpleWrapper(viewsProtocol, 'Views')
+        ]).then(function() {
+            _ = helper.injector.get('_');
+        });
+    });
+
+    beforeEach('reset stubs', function () {
+        _(viewsProtocol).methods().forEach(function (method) {
+            if (_(viewsProtocol).has(method)) {
+              viewsProtocol[method].reset();
+            }
+        }).value();
+    });
+
+    after('stop HTTP server', function () {
+        return helper.stopServer();
+    });
+
+    describe('2.0 Views', function() {
+        // should create a view (octet-stream)
+        it('should get all views', function() {
+            var views = [
+                {name: 'foo', content: 'foo', scope: 'global'},
+                {name: 'bar', content: 'bar', scope: 'global'}
+            ];
+            viewsProtocol.getAll.resolves(views);
+            return helper.request().get('/api/2.0/views')
+                .expect('Content-Type', /^application\/json/)
+                .expect(200)
+                .expect(function(res) {
+                    _(res.body).forEach(function(view, i) {
```

Comment:
Don't you need to call `.value()` here, otherwise the expect statement/function body won't get executed?